content/_index.md

```diff
@@ -6,7 +6,7 @@ type: docs
 # Acerbo datus maxime
 
 {{< columns >}}
-## MOOSE ipse furtiva
+## ipse furtiva
```

Comment:
I think that you should also capitalize these two words.  They won't match the other headers, but we can fix them later.